pkg/core/contract_listener.go

```diff
@@ -37,6 +37,12 @@ type ContractListener struct {
 	Signature string                   `ffstruct:"ContractListener" json:"signature" ffexcludeinput:"true"`
 	Topic     string                   `ffstruct:"ContractListener" json:"topic,omitempty"`
 	Options   *ContractListenerOptions `ffstruct:"ContractListener" json:"options,omitempty"`
+	State     *fftypes.JSONAny         `ffstruct:"ContractListener" json:"state,omitempty"`
+}
```

Comment:
Shouldn't this have an "Updated" timestamp?

Reply:
Sorry, this one needs to come out before this merges. I had started adding the ability for FFTM to perform updates to contract listeners... before realizing that wasn't practical as a next step due to the direct creation of listeners in EthConnect today by tokens. So I will remove this field.